low_level_platform/api/low_level_platform.h

```diff
@@ -132,6 +136,50 @@ int lf_thread_create(lf_thread_t* thread, void* (*lf_thread)(void*), void* argum
  */
 int lf_thread_join(lf_thread_t thread, void** thread_return);
 
+#define LF_SCHED_MAX_PRIORITY 99
+#define LF_SCHED_MIN_PRIORITY 0
```

Comment:
In the case of Linux, shouldn't these be the values returned by `sched_get_priority_max` and `sched_get_priority_min`?

Reply:
My thinking is that we pick these values for LF, i.e. no matter what platform you run on, setting priority of your `lf_thread_t` to 99 will be the highest.

I have now included logic for translating from this range to the range of the target platform